fjord/feedback/static/js/generic_feedback.js

```diff
@@ -160,5 +160,8 @@
                 selectHappySad(1);
             }
         }
+        if (qs.url) {
+            $('#id_url').val(qs.url);
```

Comment:
Would we want to do any data validation/fixing?

Also, if we set the value, does that trigger the input event (or whatever it is that validates the value for this field)? If not, we should make sure that gets triggered.